### ISSUE TITLE
Missing license

### ISSUE THREAD

**Issue by zimio** [Tribler/dispersy#575]
No license information is included in the project. The default copyright is all rights reserved.

**Comment by xoriole** [comment]
Dispersy is at end-of-life and no longer used.

License for Dispersy is same as Tribler and IPv8 
```GNU Lesser General Public License v3.0```
https://github.com/Tribler/tribler/blob/devel/LICENSE

**Comment by zimio** [comment]
Yes, for sure it is end of life. However this repository will be saved for posterity and someone else could find it useful.

My proposal would be to add a LICENSE file on the root of this repository to remove any ambiguity about it. The link you share shows the LICENSE file of the tribler repository which is different from this one. 

Also, maybe also add a disclaimer in the README saying how this has been deprecated in favour of IPV8.
